datasets/README.md

```diff
@@ -22,14 +25,23 @@ Thanks to using Hugging Face's `datasets` used under the hood, Flower Datasets i
 * Arrow.
 
 Create **custom partitioning schemes** or choose from the **implemented partitioning schemes**:
+
 * Partitioner (the abstract base class) `Partitioner`
 * IID partitioning `IidPartitioner(num_partitions)`
-* Natural ID partitioner `NaturalIdPartitioner`
+* Dirichlet partitioning `DirichletPartitioner(num_partitions, partition_by, alpha)`
+* InnerDirichlet partitioning `InnerDirichletPartitioner(partition_sizes, partition_by, alpha)`
+* Natural ID partitioner `NaturalIdPartitioner(partition_by)`
 * Size partitioner (the abstract base class for the partitioners dictating the division based the number of samples) `SizePartitioner`
-* Linear partitioner `LinearPartitioner`
-* Square partitioner `SquarePartitioner`
-* Exponential partitioner `ExponentialPartitioner`
-* more to come in future releases.
+* Linear partitioner `LinearPartitioner(num_partitions)`
+* Square partitioner `SquarePartitioner(num_partitions)`
+* Exponential partitioner `ExponentialPartitioner(num_partitions)`
+* more to come in the future releases (contributions are welcome).
+<p align="center">
+  <img src="./doc/source/_static/readme/comparison_of_partitioning_schemes.png" alt="Comparison of partitioning schemes."/>
+  <br>
+  <em>Comparison of Partitioning Schemes on CIFAR10</em>
+</p>
+PS: This plot was generated using a library function (see flwr_datasets.visualization package for more).
 
 # Installation
```

Comment:
I wonder if we should move this to the top of the `README.md`

Reply:
I thought about it now, and it makes sense to leave the installation below. Here's my logic: typically, you install the library once (then you can update it), so the entry point for most people is to use find the tutorials, etc. as long as the installation is somewhere low, it should be fine. 

But one can argue that the conversion rate of install can be higher if it's left at the top. But then it's a question who enters the page, if someone who doesn't know what the library does, then the install should be lower, if someone who already knows what the library should do it should be moved up.

So, I'd keep it as it is. But wdyt? if you think that it'd work better to have it at the top of the page then let's move it. 

Or maybe we can add a single line at the top that installs both images and audio, no explanation. Seems like a good compromise? (we'd also leave the full section as it is now). I'd say the same applies to the index.rst

Reply:
yes, single line makes sense to me too. Let's do it after this PR